ebpf/xdp_synproxy.c

```diff
@@ -151,9 +151,9 @@ static int syncookie_open_bpf_maps(__u32 prog_id, int *values_map_fd, int *ports
     };
     info_len = sizeof(prog_info);
 
-    err = bpf_prog_get_info_by_fd(prog_fd, &prog_info, &info_len);
+    err = bpf_obj_get_info_by_fd(prog_fd, &prog_info, &info_len);
```

Comment:
Could we test dynamically libbpf version to know which API to use ?

Reply:
there is libbpf_marjor_version API that seems to be added since libbpf version 0.8, we could use that, but what about   old distribution uses libbpf version before 0.8? not sure which version Debian 11 uses.

Reply:
Looks like it already is in 0.6.0 cf https://libbpf.readthedocs.io/en/v0.6.0/api.html